### PR TITLE
Remove irrelevant visibility options when feedback question is retrieved #4516

### LINKED ISSUE #4516: Remove irrelevant visibility options when feedback question is retrieved
Related to #4509

### PR DESCRIPTION
**Pull request by LowWeiLin**
Fixes #4516

### CODE REVIEW

File: src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java [hunk 1]
```diff
@@ -71,6 +73,8 @@ public FeedbackQuestionAttributes(String feedbackSessionName, String courseId, S
         this.showResponsesTo = showResponsesTo;
         this.showGiverNameTo = showGiverNameTo;
         this.showRecipientNameTo = showRecipientNameTo;
+        
+        removeIrrelevantVisibilityOptions();
```
Comment (kanghj) [inline src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java]: This looks like it modifies the parameters passed into the constructor?

**Comment by damithc** [comment]
Ideally, hot fixes should be done against the release branch, not master branch.

File: src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java (hunk 1 above)
Comment (LowWeiLin) [inline src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java]: Will change to copy the lists. This constructor is actually not used, so maybe i will just remove it after confirming.

**Comment by LowWeiLin** [comment]
Rebased the branch onto release, will open a PR against release branch next time.

@kanghj ready for review

**Comment by kanghj** [comment]
Looks good

**Comment by damithc** [comment]
Why still open?

**Pull request merged**